src/main/java/com/pusher/client/crypto/nacl/Poly1305.java

```diff
@@ -1370,22 +1370,22 @@ public static boolean verify(byte[] mac, byte[] m, byte[] key) {
 
         x6 += y5;
 
-        x2 += offset1;
+        x2 += OFFSET_1;
         d1 = Double.doubleToLongBits(x2);
 
-        x0 += offset0;
+        x0 += OFFSET_0;
         d0 = Double.doubleToLongBits(x0);
 
-        x4 += offset2;
+        x4 += OFFSET_2;
         d2 = Double.doubleToLongBits(x4);
 
-        x6 += offset3;
+        x6 += OFFSET_3;
         d3 = Double.doubleToLongBits(x6);
 
         f0 = d0;
 
         f1 = d1;
-        bits32 = 0xFFFFFFFFFFFFFFFFl;
```

Comment:
what does the `l` to `L` do?

Reply:
makes it more legible and less error-prone as `l` can be easily mistaken with `1`